app/services/application_service.rb

```diff
@@ -0,0 +1,7 @@
+class ApplicationService
+  class << self
+    def call(...)
+      new(...).call
+    end
```

Comment:
Not sure what this is or if it's being used

Reply:
Is this used under the hood by geocoder?

Reply:
No, just used to call services like.
`SomethingService.call(params)`